crates/noirc_evaluator/src/ssa_refactor/opt/defunctionalize.rs

```diff
@@ -250,15 +305,14 @@ fn function_id_to_field(function_id: FunctionId) -> FieldElement {
 /// Creates an apply function for the given signature and variants
 fn create_apply_function(
     ssa: &mut Ssa,
-    signature: &FunctionSignature,
-    function_ids: &[FunctionId],
+    signature: CallSignature,
+    function_ids: Vec<FunctionId>,
 ) -> FunctionId {
     assert!(!function_ids.is_empty());
     ssa.add_fn(|id| {
-        let mut function_builder = FunctionBuilder::new("apply".to_string(), id, signature.runtime);
+        let mut function_builder = FunctionBuilder::new("apply".to_string(), id, RuntimeType::Acir);
```

Comment:
Since now ACIR functions can be used inside brillig, we can just use ACIR type for apply functions